pkgs/applications/science/biology/sortmerna/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, cmake, rocksdb, rapidjson, pkgconfig, fetchFromGitHub, fetchpatch, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "sortmerna";
+  version = "3.0.3";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "biocore";
+    rev = version;
```

```suggestion
    rev = "v${version}";
```

I'm seeing tags formatted in this way
* https://github.com/biocore/sortmerna/tree/v3.0.3